.github/workflows/aws-deploy.yaml

```diff
@@ -18,24 +14,21 @@ jobs:
     name: Deploy CloudFormation templates using sceptre
     runs-on: ubuntu-latest
     needs: pre-commit
-    if: github.ref == 'refs/heads/prod'
+    if: github.ref == 'refs/heads/main'
```

Comment:
hmm, there's no `main` branch in this repo?

Reply:
Thanks for checking. I'll let @thomasyu888 create ` main` branch from `develop` once he's done cleaning up any orphaned stacks on the account. That should trigger the deployment of the the VPC template. 

I'm just trying to get these PRs merged by the end of today for Tom.